hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/embedded/ViewSingletonOp.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.utilops.embedded;
+
+import static java.util.Objects.requireNonNull;
+
+import com.hedera.services.bdd.spec.HapiSpec;
+import com.hedera.services.bdd.spec.utilops.UtilOp;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.function.Consumer;
+
+/**
+ * An operation that allows the test author to view a singleton record in an embedded state.
```

Comment:
```suggestion
 * An operation that allows the test author to view a singleton value in an embedded state.
```
I was briefly confused trying to remember what a 'singleton record [file]' is

Reply:
Changed this.